bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/component.tsx

```diff
@@ -209,66 +242,72 @@ const ChatMessageList: React.FC<ChatListProps> = ({
     ? userLoadedBackUntilPage : Math.max(totalPages - 2, 0);
   const pagesToLoad = (totalPages - firstPageToLoad) || 1;
   return (
-    <MessageListWrapper>
-      <MessageList
-        ref={messageListRef}
-        onWheel={(e) => {
-          if (e.deltaY < 0) {
-            if (isElement(contentRef.current) && followingTail) {
-              toggleFollowingTail(false)
+    [
+      <MessageListWrapper>
+        <MessageList
+          ref={messageListRef}
+          onWheel={(e) => {
+            if (e.deltaY < 0) {
+              if (isElement(contentRef.current) && followingTail) {
+                toggleFollowingTail(false)
+              }
+              setUserScrolledBack(true);
+            } else if (e.deltaY > 0) {
+              setScrollToTailEventHandler(messageListRef.current as HTMLDivElement);
             }
-          } else if (e.deltaY > 0) {
+          }}
+          onMouseUp={() => {
+            setScrollToTailEventHandler(messageListRef.current as HTMLDivElement);
+          }}
+          onTouchEnd={() => {
             setScrollToTailEventHandler(messageListRef.current as HTMLDivElement);
-          }
-        }}
-        onMouseUp={() => {
-          setScrollToTailEventHandler(messageListRef.current as HTMLDivElement);
-        }}
-        onTouchEnd={() => {
-          setScrollToTailEventHandler(messageListRef.current as HTMLDivElement);
-        }}
-      >
-        <span>
-          {
-            (userLoadedBackUntilPage)
-              ? (
-                <ButtonLoadMore
-                  onClick={() => {
-                    if (followingTail) {
-                      toggleFollowingTail(false);
+          }}
+          onScroll={handleScroll}
```

Comment:
I don't recomend user onScroll because it causes a bug when the panel is resized this event is executed and cause the scroll stop follow the tails